### ISSUE TITLE
[Bug]: LaravelLang\Config\Data\Hidden\ModelsData::__construct(): Argument #1 ($directory) must be of type string, null giv   en,

### ISSUE THREAD

**Issue by oommgg** [Laravel-Lang/common#191]
### Lang Package Name

laravel-lang/common

### Lang Package Version

6.3.0

### Laravel Framework Version

11.10.0

### PHP Version

8.2.20

### Issue description

```
LaravelLang\Config\Data\Hidden\ModelsData::__construct(): Argument #1 ($directory) must be of type string, null giv
  en, called in /Users/xxx/src/l11/vendor/laravel-lang/config/src/Services/Config.php on line 51
```

### Steps to reproduce

create a brand new laravel
```shell
laravel new L11
```
after laravel installed

```shell
composer require --dev laravel-lang/common

./composer.json has been updated
Running composer update laravel-lang/common
Loading composer repositories with package information
Updating dependencies
Lock file operations: 20 installs, 0 updates, 0 removals
  - Locking archtechx/enums (v1.0.1)
  - Locking composer/semver (3.4.0)
  - Locking dragon-code/contracts (2.23.0)
  - Locking dragon-code/pretty-array (v4.1.0)
  - Locking dragon-code/support (6.13.0)
  - Locking laravel-lang/actions (1.8.4)
  - Locking laravel-lang/attributes (2.10.7)
  - Locking laravel-lang/common (6.3.0)
  - Locking laravel-lang/config (1.4.0)
  - Locking laravel-lang/http-statuses (3.8.3)
  - Locking laravel-lang/json-fallback (2.1.0)
  - Locking laravel-lang/lang (15.5.1)
  - Locking laravel-lang/locale-list (1.4.0)
  - Locking laravel-lang/locales (2.8.0)
  - Locking laravel-lang/native-country-names (1.4.0)
  - Locking laravel-lang/native-currency-names (1.4.0)
  - Locking laravel-lang/native-locale-names (2.3.0)
  - Locking laravel-lang/publisher (16.4.0)
  - Locking laravel-lang/routes (1.1.0)
  - Locking symfony/polyfill-php81 (v1.29.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 20 installs, 0 updates, 0 removals
  - Installing dragon-code/contracts (2.23.0): Extracting archive
  - Installing symfony/polyfill-php81 (v1.29.0): Extracting archive
  - Installing dragon-code/support (6.13.0): Extracting archive
  - Installing laravel-lang/native-locale-names (2.3.0): Extracting archive
  - Installing laravel-lang/native-currency-names (1.4.0): Extracting archive
  - Installing laravel-lang/native-country-names (1.4.0): Extracting archive
  - Installing archtechx/enums (v1.0.1): Extracting archive
  - Installing laravel-lang/locale-list (1.4.0): Extracting archive
  - Installing laravel-lang/config (1.4.0): Extracting archive
  - Installing laravel-lang/locales (2.8.0): Extracting archive
  - Installing laravel-lang/routes (1.1.0): Extracting archive
  - Installing dragon-code/pretty-array (v4.1.0): Extracting archive
  - Installing composer/semver (3.4.0): Extracting archive
  - Installing laravel-lang/publisher (16.4.0): Extracting archive
  - Installing laravel-lang/lang (15.5.1): Extracting archive
  - Installing laravel-lang/json-fallback (2.1.0): Extracting archive
  - Installing laravel-lang/http-statuses (3.8.3): Extracting archive
  - Installing laravel-lang/attributes (2.10.7): Extracting archive
  - Installing laravel-lang/actions (1.8.4): Extracting archive
  - Installing laravel-lang/common (6.3.0): Extracting archive
3 package suggestions were added by new dependencies, use `composer suggest` to see details.
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

In ModelsData.php line 9:

  LaravelLang\Config\Data\Hidden\ModelsData::__construct(): Argument #1 ($directory) must be of type string, null giv
  en, called in /Users/xxx/src/l11/vendor/laravel-lang/config/src/Services/Config.php on line 51


Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

**Comment by enrymari** [comment]
I have the same issue after updating my laravel 11 project this morning.
Output from the command 'composer update':

> Loading composer repositories with package information
> Updating dependencies
> Nothing to modify in lock file
> Installing dependencies from lock file (including require-dev)
> Nothing to install, update or remove
> Generating optimized autoload files
> > Illuminate\Foundation\ComposerScripts::postAutoloadDump
> > @php artisan package:discover --ansi
> 
> In ModelsData.php line 9:
> 
>   LaravelLang\Config\Data\Hidden\ModelsData::__construct(): Argument #1 ($directory) must be of type string, null given, called
>    in /home/laravel/laravel_root/telefonia11/vendor/laravel-lang/config/src/Services/Config.php on line 51
> 
> 
> Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
> 
Is there a work-around? Thanks!

**Comment by enrymari** [comment]
Maybe the problem comes from updating laravel-lang/config (1.0.3 => 1.4.0)?

**Comment by andrey-helldar** [comment]
I apologize. I will release a hotfix in the near future.

**Comment by andrey-helldar** [comment]
The bug has been fixed in version [1.4.1](https://github.com/Laravel-Lang/config/releases/tag/1.4.1) of `laravel-lang/config` package.

Once again I apologize for the inconvenience. This was a bug that could not be detected during preliminary testing.

Thank you for your report!
